dashboard/db/migrate/20170626160941_add_storage_app_to_channel_tokens.rb

```diff
@@ -0,0 +1,5 @@
+class AddStorageAppToChannelTokens < ActiveRecord::Migration[5.0]
+  def change
+    add_column :channel_tokens, :storage_app_id, :integer, after: :channel
```

Comment:
If there is a definite schedule to migrate `storage_apps` to `channels`, I think it makes sense to name this `channel_id`. Otherwise, I think it makes more sense to name this `storage_app_id`, adding the rename as technical debt as part of renaming `storage_apps` to `channels`.

It would be confusing to have `channel_id` reference the primary key of the `storage_apps` table for an extended length of time, in my worldview.